dhcpv6/dhcpv6message.go

```diff
@@ -248,6 +248,21 @@ func (mo MessageOptions) ElapsedTime() time.Duration {
 	return 0
 }
 
+// InformationRefreshTime returns the Information Refresh Time option
+// as defined by RFC 815 Section 21.23.
+//
+// ElapsedTime returns a duration of 0 if the option is not present.
```

Comment:
This makes it impossible to differentiate packets with the explicit "0" value (that's not valid per the rfc, but it's still representable on the wire so we should be able to handle it) from packets where the option isn't supplied.

Can you make it either a pointer or return `(Duration, error)` or `(Duration, bool)` where the boolean indicates if it was found, similar to the interface of maps ?

Additionally, there's a copy-paster error, `ElapsedTime` -> `InformationRefreshTime`

Reply:
Great idea, apologies I got a bit lazy here (evidenced by the copy paste error) :)

Reply:
I wonder if we should make it consistent with the v4 one in this case:

```
func (d *DHCPv4) IPAddressRenewalTime(def time.Duration) time.Duration {
```

https://github.com/insomniacslk/dhcp/blob/master/dhcpv4/dhcpv4.go#L690

I am not sure what the best option is, just asking for opinion.

Reply:
I don't think there is a lot of value in trying to keep the interfaces the same; the protocols are different and I still haven't seen any code that really manages to handle both transparently. The current APIs are already very different

So the question is, do we have users for this api, and what do they care about ?
 * They want to get a (valid) value, anything: then the version with a default is good
 * They want to know if the option was passed, and if yes what the value is : then we want the double-return

As far as I can tell, there are no users on github of either [IPAddressRenewalTime](https://github.com/search?q=IPAddressRenewalTime&type=Code)(no hits except this lib) or [ElapsedTime](https://github.com/search?l=Go&p=3&q=dhcpv6+ElapsedTime&type=Code)(None of the hits are users of this lib). 
So maybe the simplest solution here is to not have a getter at all, until we have a client that wants it and knows what they want ?